source/unified-test-format/unified-test-format.rst

```diff
@@ -2635,22 +2635,34 @@ sections.
 Terminating Open Transactions
 `````````````````````````````
 
-Open transactions can cause tests to block indiscriminately. Test runners SHOULD
-terminate all open transactions at the start of a test suite and after each
-failed test by killing all sessions in the cluster. Using the internal
-MongoClient, execute the ``killAllSessions`` command on either the primary or,
-if connected to a sharded cluster, all mongos servers.
+Open transactions can cause tests to block indiscriminately. When connected to
+MongoDB 3.6 or later, test runners SHOULD terminate all open transactions at the
```

Comment:
Just adding "When connected to MongoDB 3.6 or later". I realized Ruby was possibly [ignoring CommandNotFound(59)](https://github.com/mongodb/mongo-ruby-driver/blob/95de4ad759a3b6959493e6d4952239a1aff7827d/spec/runners/unified/test.rb#L358) because it was testing against older servers, so this might address that edge case.

Reply:
We ran crud v1, v2 and transaction tests with the same base runner which is probably where this error handling came from.